cpp/ps0003/cpp_ps0003.cpp

```diff
@@ -36,4 +23,7 @@ int main() {
   // ternary condition-----
   std::string result = (number % 2 == 0) ? "Number is even" : "Number is odd";
   std::cout << result << std::endl;
-}
\ No newline at end of file
+
+  return 0;
+}
+// great work 😎 9/10 ⭐

```

Comment:
- [ ] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code

you can visit the [problem solution](https://onlinegdb.com/PbqxNbSVc) for more